src/components/shared/Modal.js

```diff
@@ -1,11 +1,9 @@
 // @flow
 
-import { DOM as dom, createFactory, Component, PropTypes } from "react";
+import React, { PropTypes, Component } from "react";
 import type { Children } from "react";
 import classnames from "classnames";
-import _Transition from "react-transition-group/Transition";
-const Transition = createFactory(_Transition);
-
+import Transition from "react-transition-group/Transition";
```

Comment:
I am really unsure.
Transition is used within Slide. Which is the default export.
I did a quick search but couldn't find any usage of Slide, but bc I am a total noob on your codebase this doesnot mean I didn't oversee something